devices/FreeEmgWearableDevice/conf/freeemg_analog.xml

```diff
@@ -76,6 +76,18 @@
         </group>
     </device>
 
+    <!-- Stream data as wearable message -->
+    <device type="iwear_wrapper" name="FreeEmgWearableDeviceWrapper">
+        <param extern-name="period" name="period">0.005</param><!--Device update period in seconds-->
+        <param extern-name="dataPortName" name="dataPortName">/FreeEmg/WearableData/data:o</param>
+        <action phase="startup" level="5" type="attach">
+            <paramlist name="networks">
+                <elem name="FreeEmgWearableDeviceWrapperLabel"> FreeEmgWearableDevice </elem>
+            </paramlist>
+        </action>
+        <action phase="shutdown" level="4" type="detach"/>
+    </device>
```

Comment:
This is already in a dedicated configuration file: https://github.com/ami-iit/yarp-devices-bts-freeemg/blob/c8d3ed36c64995d481883f9feb1b5305f19f12d2/devices/FreeEmgWearableDevice/conf/freeemg.xml#L80-L88

Are you sure you want to put this device also here?

Reply:
I actually don't see the wrapper in any of the other files in [`yarp-devices-bts-freeemg/devices/FreeEmgWearableDevice/conf`](https://github.com/ami-iit/yarp-devices-bts-freeemg/tree/main/devices/FreeEmgWearableDevice/conf). `freeemg_analog` streams the values as analogue data using the [`iwear_logger`](https://github.com/ami-iit/yarp-devices-bts-freeemg/blob/c8d3ed36c64995d481883f9feb1b5305f19f12d2/devices/FreeEmgWearableDevice/conf/freeemg_analog.xml#L80-L113) device, not the wearable data. Is there any other application streaming the wearable data?

Reply:
> I actually don't see the wrapper in any of the other files in [`yarp-devices-bts-freeemg/devices/FreeEmgWearableDevice/conf`](https://github.com/ami-iit/yarp-devices-bts-freeemg/tree/main/devices/FreeEmgWearableDevice/conf). `freeemg_analog` streams the values as analogue data using the [`iwear_logger`](https://github.com/ami-iit/yarp-devices-bts-freeemg/blob/c8d3ed36c64995d481883f9feb1b5305f19f12d2/devices/FreeEmgWearableDevice/conf/freeemg_analog.xml#L80-L113) device, not the wearable data. Is there any other application streaming the wearable data?

Data is streamed as wearable data in `freeemg.xml` as mentioned in my [previous comment](https://github.com/ami-iit/yarp-devices-bts-freeemg/pull/6#discussion_r1205804899), and that is actually the purpose of that file.
With this PR data is also streamed as wearable message in `freeemg_analog.xml`, which will then stream the data in both ways (analog, wearable) at the same time. Is this the intended behaviour?